94a7lc]     Headers: {"x-forwarded-for":"82.65.11.128","x-real-ip":"82.65.11.128","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWeb"}

[6y4d7h] --> GET /api/orders

[6y4d7h]     Host: logiflow.ffnancy.fr

[6y4d7h]     IP: 172.20.0.8

[6y4d7h]     Headers: {"x-forwarded-for":"82.65.11.128","x-real-ip":"82.65.11.128","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWeb"}

[94a7lc] <-- GET /api/suppliers 304 in 7ms :: [{"id":1,"name":"Fournisseur Test","contact":"Contact Principal","phone":"03.83.00.00.00","created_…

[6y4d7h] <-- GET /api/orders 304 in 10ms :: [{"id":1,"supplierId":1,"groupId":1,"plannedDate":"2025-07-02","status":"pending","notes":null,"cre…

[yafkt] --> POST /api/deliveries

[yafkt]     Host: logiflow.ffnancy.fr

[yafkt]     IP: 172.20.0.8

[yafkt]     Headers: {"x-forwarded-for":"82.65.11.128","x-real-ip":"82.65.11.128","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWeb"}

Error creating delivery: error: new row for relation "deliveries" violates check constraint "deliveries_status_check"

    at /app/node_modules/pg-pool/index.js:45:11

    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

    at async DatabaseStorage.createDelivery (file:///app/dist/index.js:5583:24)

    at async file:///app/dist/index.js:6158:24 {

  length: 359,

  severity: 'ERROR',

  code: '23514',

  detail: 'Failing row contains (2, 1, 1, 1, 2025-07-09, 1, palettes, planned, null, null, null, null, null, f, admin_local, 2025-07-11 01:39:48.588, 2025-07-11 01:39:48.588).',

  hint: undefined,

  position: undefined,

  internalPosition: undefined,

  internalQuery: undefined,

  where: undefined,

  schema: 'public',

  table: 'deliveries',

  column: undefined,

  dataType: undefined,

  constraint: 'deliveries_status_check',

  file: 'execMain.c',

  line: '2023',

  routine: 'ExecConstraints'

}

[yafkt] <-- POST /api/deliveries 500 in 14ms :: {"message":"Failed to create delivery"}

[yafkt] ERROR Response: 500 Internal Server Error

